src/index.js

```diff
@@ -78,12 +80,16 @@ render(
     <ReactReduxFirebaseProvider {...rrfProps}>
       <Router>
         <Switch>
+          <Route
+            path="/account/edit"
+            component={withLoadedUsers(EditProfilePage)}
```

Comment:
I'd prefer a component wrapping the routes here. Would look like
```
<Switch>
  ...
  <LoadedUsers>
    <Switch>
      <Route path="/account/edit" />
      <Route path="/" component={withLoadedUsers(App)} />
    </Switch>
  </LoadedUsers>
</Switch>
```

That way we have sections clearly defined in the routing file.

Reply:
great idea, I totally approve